### PR TITLE
[testing][e2e] IngressNginxController was enabled

### PR DESCRIPTION
**Pull request by apolovov**
## Description
e2e tests improvements:
* IngressNginxController was added

## Why do we need it, and what problem does it solve?
It is a good idea to seek regressions related to ingress-nginx and istio modules.

## What is the expected result?
ingress-nginx module is covered by e2e tests.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: testing
type: chore
summary: The `ingress-nginx` module was enabled for e2e testing.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/openstack'...

File: testing/cloud_layouts/AWS/WithoutNAT/resources.yaml [hunk 1]
```diff
@@ -19,10 +19,14 @@ spec:
   nodeTemplate:
     labels:
       node-role.deckhouse.io/system: ""
+      node-role.deckhouse.io/frontend: ""
     taints:
     - effect: NoExecute
       key: dedicated.deckhouse.io
       value: system
+    - effect: NoExecute
```
Comment (duckhawk) [inline testing/cloud_layouts/AWS/WithoutNAT/resources.yaml:27]: But with this taints any system component without frontend toleration can't get to system nodes (like prom, trickster, etc)

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/openstack'...

**Comment by fl64** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.23 4193309429 4193309429-2-con-1-23 /sys/deckhouse-oss/install:pr2870 centos@3.124.138.43

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by fl64** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.24 4191527556 4191527556-1-con-1-24 /sys/deckhouse-oss/install:pr2870
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.24 4191751858 4191751858-1-con-1-24 /sys/deckhouse-oss/install:pr2870
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.24 4191751858 4191751858-1-con-1-24 /sys/deckhouse-oss/install:pr2870
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.23 4191940842 4191940842-1-con-1-23 /sys/deckhouse-oss/install:pr2870
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.23 4192267114 4192267114-1-con-1-23 /sys/deckhouse-oss/install:pr2870
/e2e/abort aws;WithoutNAT;containerd;1.23 4193309429 4193309429-2-con-1-23 /sys/deckhouse-oss/install:pr2870 centos@3.124.138.43
/e2e/abort openstack;Standard;containerd;1.23 4194324131 4194324131-1-con-1-23 /sys/deckhouse-oss/install:pr2870 debian@45.94.123.226 
/e2e/abort static;Static;containerd;1.24 4201265621 4201265621-3-con-1-24 /sys/deckhouse-oss/install:pr2870 astra@192.168.199.69
/e2e/abort aws;WithoutNAT;containerd;1.24 4201265789 4201265789-2-con-1-24 /sys/deckhouse-oss/install:pr2870 centos@3.125.96.91
/e2e/abort static;Static;containerd;1.24 4202077992 4202077992-1-con-1-24 /sys/deckhouse-oss/install:pr2870
/e2e/abort static;Static;containerd;1.24 4202077992 4202077992-1-con-1-24 /sys/deckhouse-oss/install:pr2870 
/e2e/abort openstack;Standard;containerd;1.24 4202078464 4202078464-1-con-1-24 /sys/deckhouse-oss/install:pr2870

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by fl64** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.24 4191751858 4191751858-1-con-1-24 /sys/deckhouse-oss/install:pr2870

**Comment by fl64** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.24 4191751858 4191751858-1-con-1-24 /sys/deckhouse-oss/install:pr2870

**Comment by fl64** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.23 4191940842 4191940842-1-con-1-23 /sys/deckhouse-oss/install:pr2870

**Comment by fl64** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.23 4192267114 4192267114-1-con-1-23 /sys/deckhouse-oss/install:pr2870

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by fl64** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.23 4193309429 4193309429-2-con-1-23 /sys/deckhouse-oss/install:pr2870 centos@3.124.138.43

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by fl64** [comment]
/e2e/abort openstack;Standard;containerd;1.23 4194324131 4194324131-1-con-1-23 /sys/deckhouse-oss/install:pr2870 debian@45.94.123.226

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by fl64** [comment]
/e2e/abort static;Static;containerd;1.24 4201265621 4201265621-3-con-1-24 /sys/deckhouse-oss/install:pr2870 astra@192.168.199.69

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by fl64** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.24 4201265789 4201265789-2-con-1-24 /sys/deckhouse-oss/install:pr2870 centos@3.125.96.91

**Comment by fl64** [comment]
/e2e/abort static;Static;containerd;1.24 4202077992 4202077992-1-con-1-24 /sys/deckhouse-oss/install:pr2870

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by fl64** [comment]
/e2e/abort static;Static;containerd;1.24 4202077992 4202077992-1-con-1-24 /sys/deckhouse-oss/install:pr2870

**Comment by fl64** [comment]
/e2e/abort openstack;Standard;containerd;1.24 4202078464 4202078464-1-con-1-24 /sys/deckhouse-oss/install:pr2870

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/openstack'...

**Comment by fl64** [comment]
/e2e/abort static;Static;containerd;1.24 4202480336 4202480336-1-con-1-24 /sys/deckhouse-oss/install:pr2870 astra@192.168.199.207

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/aws'...

**Comment by fl64** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.24 4202979985 4202979985-1-con-1-24 /sys/deckhouse-oss/install:pr2870 centos@18.198.45.211

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @fl64. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by z9r5** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.24 4201265789 4201265789-2-con-1-24 /sys/deckhouse-oss/install:pr2870 centos@3.125.96.91

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by z9r5** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.24 4202078039 4202078039-1-con-1-24 /sys/deckhouse-oss/install:pr2870 centos@18.156.39.66

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by z9r5** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.24 4202979985 4202979985-1-con-1-24 /sys/deckhouse-oss/install:pr2870 centos@18.198.45.211

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Review by duckhawk — APPROVED** [review]
LGTM

File: testing/cloud_layouts/AWS/WithoutNAT/resources.yaml (hunk 1 above)
Comment (fl64) [reply testing/cloud_layouts/AWS/WithoutNAT/resources.yaml]: I did it in other way

**Pull request merged**